plugins/CoreHome/angularjs/period-selector/period-selector.controller.js

```diff
@@ -185,6 +185,8 @@
         }
 
         function propagateNewUrlParams(date, period) {
+            piwik.updateDateInTitle(date, period);
```

Comment:
Noticed this isn't really necessary, could probably remove it. Below since `$location.search()` is called, `updatePeriodParamsFromUrl()` ends up being called in `piwikService`. It would be useful outside of an angular context, but that only happens in the embedded pages, and since those are shown in iframes, the document title isn't really that useful

Reply:
Ok! I wasn't certain about if or when angular might not be rendering the page, so I added this here to be safe. It sounds like it is unnecessary, so I will go ahead and remove it.